Common/UnitDefinitions/RotationalAcceleration.json

```diff
@@ -41,6 +41,18 @@
           "Abbreviations": [ "rpm/s" ]
         }
       ]
+    },
+    {
+      "SingularName": "RevolutionPerSecondPerSecond",
```

Comment:
You named it "revolutions per second squared" in the description. Should maybe the unit also be named that? What is more natural in your opinion?
I see the other units are named `PerSecondSquared` too, so it would be more consistent I think.

Reply:
Agreed, I'll change it to `RevolutionPerSecondSquared` in the next commit